ethcore/res/ethereum/expanse.json

```diff
@@ -15,11 +15,11 @@
 				"difficultyHardforkTransition": "0x59d9",
 				"difficultyHardforkBoundDivisor": "0x0200",
 				"bombDefuseTransition": "0x30d40",
-				"eip150Transition": "0x7fffffffffffffff",
-				"eip155Transition": "0x7fffffffffffffff",
-				"eip160Transition": "0x7fffffffffffffff",
-				"eip161abcTransition": "0x7fffffffffffffff",
-				"eip161dTransition": "0x7fffffffffffffff"
+				"eip150Transition": "0x927C0",
+				"eip155Transition": "0x927C0",
```

Comment:
Sure, I get what you are saying, however Expanse uses a completely different network identifier so nodes dont even have the ability to talk to each other and fail at the handshake. I suppose people could copy over a raw tx and try to replay those txs on EXP but its highly unlikely any EXP addresses are also used for ETH.